fiftyone/utils/cvat.py

```
@@ -6498,14 +6498,21 @@ def _create_detection_shapes(
+                    w, h = round(w * frame_width), round(h * frame_height)
+                    xbr, ybr = xtl + w, ytl + h
 
                     # -1 to convert from CVAT indexing
-                    rle = HasCVATBinaryMask._mask_to_cvat_rle(det.mask)
+                    mask = det.mask
+                    if w != mask_width or h != mask_height:
+                        mask = etai.resize(
+                            mask.astype("uint8"), width=w, height=h
+                        )
+
+                    rle = HasCVATBinaryMask._mask_to_cvat_rle(mask)
```

Pylint (3.3.7)

[warning] Access to a protected member _mask_to_cvat_rle of a client class

(W0212)